cxx-qt-gen/src/parser/mod.rs

```diff
@@ -165,4 +183,17 @@ mod tests {
         let parser = Parser::from(module);
         assert!(parser.is_err());
     }
+
+    #[test]
+    fn test_parser_from_error_no_attribute() {
+        let module: ItemMod = tokens_to_syn(quote! {
+            mod ffi {
+                extern "Rust" {
+                    fn test();
+                }
+            }
+        });
+        let parser = Parser::from(module);
+        assert!(parser.is_err());
```

Comment:
Should this test more specifically what kind of error it is?

Reply:
We can't easily as the syn Error's are just a span and a string, so just checking for is_err is enough for now.

Reply:
okay